railgun/test/integration/suite_test.go

```diff
@@ -88,8 +88,11 @@ func TestMain(m *testing.M) {
 		}
 		go waitForExistingClusterReadiness(ctx, cluster, existingClusterName, ready)
 	} else {
-		// create a new cluster for tests
-		config := ktfkind.ClusterConfigurationWithKongProxy{EnableMetalLB: true}
+		// configure and deploy a new cluster for tests
+		config := ktfkind.ClusterConfigurationWithKongProxy{
+			EnableMetalLB: true,
+			DBMode:        os.Getenv("KONG_DATABASE"),
```

Comment:
optional

I suggest defining a global

```go
var (
  IsDBMode = os.Getenv("KONG_DATABASE")
  // optionally other test configuration knobs
)
```

so that `os.Getenv`s are not scattered across the test codebase

Reply:
Sounds good I'll do this :+1:

Reply:
39cc96b2a79c2e885918ebb32ec9e77d2717d699